basiccompose.txt

```diff
@@ -0,0 +1,38 @@
+version: '2'
```

Comment:
Please change the extension for this file to .yml so it can be used as a base for other users